<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Usaha</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="iDUSAHALabel">
       <property name="text">
        <string>ID USAHA</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="iDUSAHALineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nAMAUSAHALabel">
       <property name="text">
        <string>NAMA USAHA</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nAMAUSAHALineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lOKASILabel">
       <property name="text">
        <string>LOKASI</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lOKASILineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="tAHUNBERDIRILabel">
       <property name="text">
        <string>TAHUN BERDIRI</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="tAHUNBERDIRISpinBox">
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>2100</number>
       </property>
       <property name="value">
        <number>2025</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="jUMLAHBURUHLabel">
       <property name="text">
        <string>JUMLAH BURUH</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="jUMLAHBURUHSpinBox">
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="pENDAPATANLabel">
       <property name="text">
        <string>PENDAPATAN</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="pENDAPATANLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>SIMPAN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>HAPUS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>BERSIH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>UBAH</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="EditCari"/>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>id_usaha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nama_usaha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>lokasi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>tahun_berdiri</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>jumlah_buruh</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pendapatan_bulanan</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
